src/Mappers/TransactionMapper.php

```diff
@@ -263,6 +263,20 @@ private static function getField(BaseTransaction $transaction, \DOMElement $elem
         return $fieldElement->textContent;
     }
 
+    private static function getFieldAsMoneyObject(
```

Comment:
I think that the `Object` postfix is unnecessary.

Reply:
The method is a great addition, though.

Reply:
✅  Removed the postfix.